src/main/java/org/hibernate/models/internal/dynamic/DynamicAnnotationUsage.java

```diff
@@ -109,4 +118,46 @@ private static <A extends Annotation> Map<String, Object> extractBaselineValues(
 		}
 		return values;
 	}
+
+	private static <A extends Annotation> Map<String, Object> extractBaselineValues(
```

Comment:
We can delete the old version of the `extractBaselineValues` method, this should replace it.